app/controllers/api/account.php

```diff
@@ -109,7 +112,8 @@
                 'email' => $email,
                 'emailVerification' => false,
                 'status' => true,
-                'password' => Auth::passwordHash($password, Auth::DEFAULT_ALGO, Auth::DEFAULT_ALGO_OPTIONS),
+                'passwordHistory' => $passwordHistory > 0 ? [$password] : [],
```

Comment:
Let's group all password related attributes together in all controllers.

Reply:
done in dictionary PR